data/extension/init/templates/rust/envoy.access_loggers/default/src/logger.rs

```diff
@@ -67,11 +65,9 @@ impl<'a> AccessLogger for SampleAccessLogger<'a> {
         // Update stats
         self.stats.log_entries_total().inc()?;
 
-        let now: DateTime<Local> = self.clock.now()?.into();
-
         log::info!(
-            "logging at {} with config: {:?}",
-            now.format("%+"),
+            "logging at {:?} with config: {:?}",
+            self.clock.now(),
```

Comment:
this still uses the api, just doesn't bother converting it to a more readable date